sdk/cognitivelanguage/ai-language-conversations/package.json

```diff
@@ -11,6 +11,8 @@
     "@azure/core-client": "^1.5.0",
     "@azure/core-auth": "^1.3.0",
     "@azure/core-rest-pipeline": "^1.8.0",
+    "@azure/core-tracing": "1.0.0",
+    "@azure/identity": "^2.1.0",
```

Comment:
identity shouldn't be a runtime dep, core-auth should be enough in this list.

Reply:
Actually, identity is only used in AAD auth sample, should it still be under dev deps>

Reply:
Yes, deps needed by samples and tests should be in devDeps.